### PR TITLE
Update to .NET Core 1.1.12 and 1.0.15

### PR DESCRIPTION
**Pull request by JunTaoLuo**
March update

### CODE REVIEW

File: README.aspnetcore-build-nightly.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ The [.NET Core Docker samples](https://github.com/dotnet/dotnet-docker/blob/mast
 
 # Linux amd64 tags
 
-- [`1.1.11-1.1.12-jessie`, `1.1.11-1.1.12`, `1.1`, `1` (*1.1/jessie/sdk/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/jessie/sdk/Dockerfile)
+- [`1.1.12-1.1.13-jessie`, `1.1.12-1.1.13`, `1.1`, `1` (*1.1/jessie/sdk/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/jessie/sdk/Dockerfile)
```
Comment (JunTaoLuo) [inline README.aspnetcore-build-nightly.md]: @MichaelSimons are these tags still formatted correctly or should they be update to just the sdk version `1.1.13` instead?

Comment (MichaelSimons) [reply README.aspnetcore-build-nightly.md]: I think they should remain as is.

File: 1.1/nanoserver-sac2016/sdk/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # escape=`
-FROM mcr.microsoft.com/dotnet/core/sdk:1.1.12-nanoserver-sac2016
+FROM mcr.microsoft.com/dotnet/core/sdk:1.1.13-nanoserver-sac2016
```
Comment (MichaelSimons) [inline 1.1/nanoserver-sac2016/sdk/Dockerfile]: The nanoserver-sac2016 images were deprecated from the .NET Core repos and therefore aren't being serviced any longer - https://github.com/dotnet/dotnet-docker/pull/957.

Comment (muratg) [reply 1.1/nanoserver-sac2016/sdk/Dockerfile]: @JunTaoLuo Let's skip the images on our end as well (the ones that are based on`nanoserver-sac2016`).  Usage of these are extremely low, and we don't have a supported base image anyway.

Comment (JunTaoLuo) [reply 1.1/nanoserver-sac2016/sdk/Dockerfile]: Sounds good.

Comment (JunTaoLuo) [reply 1.1/nanoserver-sac2016/sdk/Dockerfile]: I removed all nanoserver-sac2016 images and sections in readmes. Is that all we need to do?

File: README.aspnetcore-build.md [hunk 3]
```diff
@@ -12,12 +12,7 @@ The [.NET Core Docker samples](https://github.com/dotnet/dotnet-docker/blob/mast
 
 # Linux amd64 tags
 
-- [`1.1.11-1.1.12-jessie`, `1.1.11-1.1.12`, `1.1`, `1` (*1.1/jessie/sdk/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/jessie/sdk/Dockerfile)
```
Comment (MichaelSimons) [inline README.aspnetcore-build.md]: I see at least one reference in this readme to Windows Containers later on that should be cleaned up.

```
(Windows containers) Start a container from your image, get its assigned IP address, and then open your browser to the IP address of the container on port 80. To see console output, attach to the running container or use docker logs.
```

**Comment by JunTaoLuo** [comment]
@MichaelSimons how do I disable the Windows sac2016 1 Build? It still shows up as a check on github.

**Comment by MichaelSimons** [comment]
It looks like https://github.com/aspnet/aspnet-docker/blob/master/build-pipeline/windows-pipeline.groovy needs to be removed.  You will also need to update the official build to remove the Windows legs.

**Comment by MichaelSimons** [comment]
The build leg that needs to be removed is https://github.com/aspnet/aspnet-docker/blob/master/build-pipeline/aspnet-docker-windows-images.json

**Pull request merged**